CHANGELOG.md

```diff
@@ -55,6 +55,7 @@
 * [ENHANCEMENT] Ingester, Distributor: added experimental support for rejecting push requests received via gRPC before reading them into memory, if ingester or distributor is unable to accept the request. This is activated by using `-ingester.limit-inflight-requests-using-grpc-method-limiter` for ingester, and `-distributor.limit-inflight-requests-using-grpc-method-limiter` for distributor. #5976 #6300
 * [ENHANCEMENT] Query-frontend: return warnings generated during query evaluation. #6391
 * [ENHANCEMENT] Server: Add the option `-server.http-read-header-timeout` to enable specifying a timeout for reading HTTP request headers. It defaults to 0, in which case reading of headers can take up to `-server.http-read-timeout`, leaving no time for reading body, if there's any. #6517
+* [ENHANCEMENT] Ingester: add new per-tenant `cortex_ingester_local_limits` metric to expose the local per-tenant series limits seen at each ingester. #6403
```

Comment:
i'd mention the name and value of the label users should look for

Reply:
Updated in https://github.com/grafana/mimir/pull/6403/commits/189d777fffd9b95bda37290d152fbe33216f3a7f